bundles/org.openhab.core.io.rest.core/src/test/java/org/openhab/core/io/rest/core/internal/item/MetadataSelectorMatcherTest.java

```diff
@@ -53,21 +53,21 @@
 
     @BeforeEach
     public void setup() throws Exception {
-        when(configDescriptionRegistry.getConfigDescriptions(null)).thenReturn(mockConfigDescriptions());
+        when(metadataRegistry.getAll()).thenReturn(mockMetadata());
         when(metadataRegistry.isInternalNamespace(anyString())).thenReturn(false);
 
-        matcher = new MetadataSelectorMatcher(metadataRegistry, configDescriptionRegistry);
+        matcher = new MetadataSelectorMatcher(metadataRegistry);
```

Comment:
Private property `configDescriptionRegistry` can be removed completely from this test.